backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc

```diff
@@ -12,7 +12,6 @@ This process depends on whether the etcd member is unhealthy because the machine
 [NOTE]
 ====
 If you have lost the majority of your control plane hosts (also known as the master hosts), leading to etcd quorum loss, then you must follow the disaster recovery procedure to xref:../../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore to a previous cluster state] instead of this procedure.
-=======
```

Comment:
This looks like a note termination gone wrong. It should probably be ====

Reply:
It is not--expand below the change.

Reply:
Gotcha.